Dockerfile

```diff
@@ -1,30 +1,21 @@
-# This dockerfile expects Connector jars to have been built under a `connectors` directory
-#
-FROM alpine as builder
+FROM strimzi/kafka:latest-kafka-2.6.0 as builder
 
-RUN apk update
-RUN apk --no-cache add curl
-
-RUN curl -L "https://downloads.apache.org/kafka/3.4.1/kafka_2.12-3.4.1.tgz" -o kafka.tgz
-RUN mkdir /opt/kafka \
-    && tar -xf kafka.tgz -C /opt/kafka --strip-components=1
-
-FROM ibmjava:11
+FROM ibmjava:8-jre
 
 RUN addgroup --gid 5000 --system esgroup && \
     adduser --uid 5000 --ingroup esgroup --system esuser
 
 COPY --chown=esuser:esgroup --from=builder /opt/kafka/bin/ /opt/kafka/bin/
 COPY --chown=esuser:esgroup --from=builder /opt/kafka/libs/ /opt/kafka/libs/
-COPY --chown=esuser:esgroup --from=builder /opt/kafka/config/ /opt/kafka/config/
+COPY --chown=esuser:esgroup --from=builder /opt/kafka/config/connect-distributed.properties /opt/kafka/config/
+COPY --chown=esuser:esgroup --from=builder /opt/kafka/config/connect-log4j.properties /opt/kafka/config/
 RUN mkdir /opt/kafka/logs && chown esuser:esgroup /opt/kafka/logs
-
-COPY --chown=esuser:esgroup target/kafka-connect-*-jar-with-dependencies.jar /opt/connectors/
+COPY --chown=esuser:esgroup target/kafka-connect-mq-source-2.0.0-jar-with-dependencies.jar /opt/kafka/libs/
 
 WORKDIR /opt/kafka
 
 EXPOSE 8083
 
 USER esuser
 
-ENTRYPOINT ["./bin/connect-distributed.sh", "config/connect-distributed.properties"]
+ENTRYPOINT ["./bin/connect-distributed.sh", "config/connect-distributed.properties"]
```

Comment:
Its better to revert this file

Reply:
Could you please see if you could add in the new lines so that it doesn't show the files in the PR comparison.